admin/services/data-access/sql.pool.service.js

```diff
@@ -29,10 +29,10 @@ let pool = null
 
 const sqlPoolService = {}
 
-  /**
-   * Initialise the connection pool.  Called once per application instance
-   * @return {Promise}
-   */
+/**
+ * Initialise the connection pool.  Called once per application instance
+ * @return {Promise}
```

Comment:
Is the return needed here?
this method doesn't seem like it's returning anything.

Reply:
removed